arches/app/media/js/views/components/search/advanced-search.js

```diff
@@ -50,6 +50,7 @@ define([
                     card.nodes = _.filter(response.nodes, function(node) {
                         return node.nodegroup_id === card.nodegroup_id;
                     });
+                    card.nodeStrConcat = card.nodes.map(node => node.name).join(' ');
```

Comment:
can this name be changed to something more descriptive/readable? Something similar to `nodeNames` or `concatenatedNodeNames`?

Reply:
👍 just pushed the commit